projects/kserve/testing/config.yaml

```diff
@@ -420,6 +426,10 @@ tests:
       concurrency: 16
       plugin: caikit_client_plugin
       interface: grpc
+    longevity:
+      enabled: false
```

Comment:
I don't like this flag as it is now, because it's not mutually exclusive with the single/multi tests.

Can you change:
```
tests:
  e2e:
    perf_mode: true
```
and rename this `mode`, so that the mode can be `single`, `multi` or `longevity` (and adapt the rest of the code, shouldn't be much)